.travis.yml

```diff
@@ -9,7 +9,9 @@ services:
 
 install: yarn
 
-script: ./go.sh yarn test
+script:
+  - ./go.sh yarn test
+  - ./go.sh e2e
```

Comment:
See comment right at the end, but I think it might be better to just directly do `./bin/e2e.sh` as the second command here.